### PR TITLE
feat: port addTokensToVault action to SDK

### PR DESCRIPTION
**Pull request by tbiedukhin**
Part of https://github.com/metaplex-foundation/js/issues/109

### CODE REVIEW

**Comment by b2kdaman** [comment]
lgtm

**Comment by tbiedukhin** [comment]
@austbot Please review

File: src/actions/addTokensToVault.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+interface Token2Add {
+  tokenAccount: PublicKey;
+  tokenMint: PublicKey;
+  amount: BN;
+}
+
+interface AddTokensToVaultParams {
+  connection: Connection;
+  wallet: Wallet;
+  vaultPub: PublicKey;
```
Comment (kurpav) [inline src/actions/addTokensToVault.ts:25]: if we use typescript we can omit type hints in the field name

File: test/actions/addTokensToVault.test.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import { AccountLayout, MintLayout } from '@solana/spl-token';
+import { Keypair, sendAndConfirmTransaction } from '@solana/web3.js';
+
+import { Connection, NodeWallet } from '../../src';
+import { FEE_PAYER, pause, NETWORK } from '../utils';
+import { CreateMint, CreateTokenAccount, MintTo } from '../../src/programs';
+import { addTokensToVault, createExternalPriceAccount, createVault } from '../../src/actions';
+
+describe('addTokensToVault action', () => {
+  const connection = new Connection(NETWORK);
```
Comment (kurpav) [inline test/actions/addTokensToVault.test.ts:12]: please use localnet for tests
https://github.com/metaplex-foundation/js/blob/main/test/actions/createExternalPriceAccountLocal.test.ts

**Review by kurpav — CHANGES_REQUESTED** [review]

File: src/actions/addTokensToVault.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  tokenStoreAccount: PublicKey;
+  tokenMint: PublicKey;
+}
+
+export const addTokensToVault = async ({
+  connection,
+  wallet,
+  vaultPub,
+  nfts,
+}: AddTokensToVaultParams): Promise<{ safetyDepositTokenStores: SafetyDepositTokenStore[] }> => {
```
[the remark this answers is not in the document]
Comment (tbiedukhin) [reply src/actions/addTokensToVault.ts]: upd

File: src/actions/addTokensToVault.ts (hunk 1 above)
Comment (tbiedukhin) [reply src/actions/addTokensToVault.ts]: upd

File: test/actions/addTokensToVault.test.ts (hunk 2 above)
Comment (tbiedukhin) [reply test/actions/addTokensToVault.test.ts]: upd

File: test/actions/addTokensToVault.test.ts [hunk 4]
```diff
@@ -1,90 +1,86 @@
...
 
-        await sendAndConfirmTransaction(
-          connection,
-          Transaction.fromCombined([createMintTx, createTokenAccountTx, mintToTokenAccountTx]),
-          [FEE_PAYER, mint, tokenAccount, wallet.payer],
-          {
-            commitment: 'confirmed',
-          },
-        );
+      const mintToTokenAccountTx = new MintTo(
```
Comment (kurpav) [inline test/actions/addTokensToVault.test.ts:51]: I think we can shorten this code by using
https://github.com/metaplex-foundation/js/blob/main/src/actions/shared/mint.ts
Can you check it, please?

Comment (tbiedukhin) [reply test/actions/addTokensToVault.test.ts]: Looks good, thanks

**Review by kurpav — APPROVED** [review]

**Pull request merged**